packages/scaffold-config/src/frameworks.ts

```diff
@@ -43,7 +43,10 @@ export const FRONTEND_FRAMEWORKS = [
       module.exports = {
         component: {
           devServer,
-        }
+          devServerConfig: {
```

Comment:
We definitely need a better system for all these configs other than copy paste. That said, we had string interpolation before and it was not ideal, so I'm okay with this for now. Let's try and think of something more scalable and less boilerplate-y in the future, though.

Reply:
I agree. 
The usual best practice solution is to as little code in those files as possible.
We should add all this in the defaults. This way we can improve the defaults as we improve the rest. 

Another argument to not scaffold this line or make it a required file is that... well, we have no use for it internally do we?